src/components/soknadpanel/types.ts

```diff
@@ -0,0 +1,98 @@
+export enum TagTyper {
```

Comment:
![mother-of-god](https://user-images.githubusercontent.com/1507032/157265663-4fc1ae7e-c714-475a-b6c4-451f0d0a3bff.gif)